server/migrations/helpers.js

```diff
@@ -0,0 +1,50 @@
+function calculatePreferences(user) {
+  const updatedPreferences = {
+    config: '',
+    sort: 0,
+    star: user?.preferences?.star || [],
```

Comment:
will we ever not have a user here?

```suggestion
    star: user.preferences?.star || [],
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241